src/stories/LineVis.stories.tsx

```diff
@@ -90,6 +94,23 @@ CustomAbscissas.args = {
   },
 };
 
+export const SemilogX = Template.bind({});
+
+SemilogX.args = {
```

Comment:
Some naming/capitalisation inconsistencies between stories (with or without `ScaleFor`, SemiLog` with upper or lower case `L`).

Reply:
In fact, I wrote `Semilog` (which is [a whole word](https://en.wiktionary.org/wiki/semilog)) to describe the plot. While `SymLog` (for Symmetric Log) is the scale used for the plot.

Reply:
Should I rather use `[Log|SymLog]ScaleFor[X/Y]` in every story name ?

Reply:
Instead, I use another level of hierarchy to put all scale-related stories in their own folder